docs/administrator.md

```diff
@@ -507,6 +507,33 @@ A secret can be pre-provisioned in different ways:
 * Automatically provisioned via a custom K8s controller like
   [kube-aws-iam-controller](https://github.com/mikkeloscar/kube-aws-iam-controller)
 
+## Sidecars for Postgres clusters
```

Comment:
Could you add a mention and link from the [user docs](https://github.com/zalando/postgres-operator/blob/master/docs/user.md#sidecar-support) to this section?